src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -175,18 +175,23 @@ public void onActivityCreated(Activity activity, Bundle bundle) {
 
         @Override
         public void onActivityStarted(Activity activity) {
+            ConnectivityIndicator.start(mContext);
```

Comment:
It will be initialized in any case, if its onActivityResumed or onActivityStarted. The difference is that onActivityStarted will be called less times and the item will be initialized only once in any case